diff --git a/src/components/SearchFilters/SearchFilters.js b/src/components/SearchFilters/SearchFilters.js
index 64d9abce..40faa01e 100644
--- a/src/components/SearchFilters/SearchFilters.js
+++ b/src/components/SearchFilters/SearchFilters.js
@@ -6,12 +6,7 @@ import classNames from 'classnames';
 import { withRouter } from 'react-router-dom';
 import omit from 'lodash/omit';
 
-import { 
-	SelectSingleFilter, 
-	SelectMultipleFilter, 
-	PriceFilter,
-	NamedLink
-} from '../../components';
+import { SelectSingleFilter, SelectMultipleFilter, PriceFilter } from '../../components';
 import routeConfiguration from '../../routeConfiguration';
 import { createResourceLocatorString } from '../../util/routes';
 import { propTypes } from '../../util/types';
@@ -173,11 +168,6 @@ const SearchFiltersComponent = props => {
     />
   ) : null;
 
-	/*
-	 * This implement the redirection toward Recommand Deal page
-	 */
-	const recommandDealActive = true;
-
   const toggleSearchFiltersPanelButtonClasses =
     isSearchFiltersPanelOpen || searchFiltersPanelSelectedCount > 0
       ? css.searchFiltersPanelOpen
@@ -215,17 +205,7 @@ const SearchFiltersComponent = props => {
 
       {hasNoResult ? (
         <div className={css.noSearchResults}>
-					<p>
-	          <FormattedMessage id="SearchFilters.noResults" />
-					</p>
-					{recommandDealActive? (
-						<p className={css.recommendDeal}>
-							<FormattedMessage id="SearchFilters.recommendDealLabel" />
-							<NamedLink name="RecommendDealPage"> 
-								&nbsp;<FormattedMessage id="SearchFilters.recommedDealLink" />
-							</NamedLink>
-						</p>
-					)	: null}
+          <FormattedMessage id="SearchFilters.noResults" />
         </div>
       ) : null}
 
diff --git a/src/translations/en.json b/src/translations/en.json
index cdec345c..6e0878f0 100644
--- a/src/translations/en.json
+++ b/src/translations/en.json
@@ -632,8 +632,6 @@
   "Reviews.bannedUserDisplayName": "Banned user",
   "SearchFilters.amenitiesLabel": "Amenities",
   "SearchFilters.regularlyOpenOnLabel": "Regularly open on",
-  "SearchFilters.recommendDealLabel": "Earn a gift card to use over 2.000 shops! Get 15€ for recommending a pub and celebrate there.",
-  "SearchFilters.recommendDealLink": "Learn More…",
   "SearchFilters.groupSizeLabel": "Group size",
   "SearchFilters.listingTypeLabel": "Type",
   "SearchFilters.filtersButtonLabel": "Filters",
